nautobot/extras/views.py

```diff
@@ -2589,38 +2594,56 @@ def get_extra_context(self, request, instance):
 #
 
 
-class SecretListView(generic.ObjectListView):
+class SecretUIViewSet(
```

Comment:
I noticed that there is no way to add a `secret` to a `secret group` from the secret side. I wonder if it is a miss or an intentional thing.

Reply:
Since assigning a secret to a group requires specifying additional attributes on the many-to-many through table we'd have to figure out what the UI would need to look like.